src/plugins/Utils.ts

```diff
@@ -234,6 +241,28 @@ class Utils {
 
         return monitoringData;
     }
+
+    static addApplicationTags(applicationTags: any): void {
+        for (const key of Object.keys(process.env)) {
+            if (key.startsWith(envVariableKeys.THUNDRA_APPLICATION_TAG_PROP_NAME_PREFIX)) {
+                try {
+                    const propsKey = key.substring(envVariableKeys.THUNDRA_APPLICATION_TAG_PROP_NAME_PREFIX.length);
+                    const propsValue = process.env[key];
+                    if (isNaN(parseInt(propsValue, 10))) {
```

Comment:
is it possible to detect and get also decimal numbers (in addition to boolean, integer and string) as application tags?

Reply:
Ok, right now we can parse float values too. I added test case for float numbers.